data/gateways.json

```
@@ -69,6 +69,10 @@
     {
       "id": "celo",
       "address": "0xe432150cce91c13a887f7D836923d5597adD8E31"
+    },
+    {
+      "id": "kava",
+      "address": "0xe432150cce91c13a887f7D836923d5597adD8E31"
```

```suggestion
      "address": "0xC8D18F85cB0Cee5C95eC29c69DeaF6cea972349c"
```
registered gateway address is different

---

damn. didnt copy over i think.